technical-reports/color/index.html

```diff
@@ -20,12 +20,13 @@
             name: 'Ayesha Mazrana (Mazumdar)',
             url: 'https://twitter.com/AyeshaKMaz',
           },
-          { name: 'Kathleen McMahon', url: 'https://twitter.com/resource11' },
+          { name: 'Kathleen McMahon', url: 'https://kathleenmcmahon.dev' },
```

Comment:
Hey @ilikescience, we updated Adekunle's and Ayeha's urls in previous commits, yet decided to make that change spec-wide in a separate commit (per @kaelig). Now that we're updating here, will you change Adekunle's url to https://www.adekunleoduye.com/ and Ayesha's to https://github.com/ayeshakmaz please?

Reply:
will do!